src/services/stac/index.tsx

```diff
@@ -99,6 +99,103 @@ export const getStacItems = async (
   };
 };
 
+export const fetchFavouritedItems = async (collectionId: string): Promise<string[]> => {
+  const workspace = (await getActiveWorkspace()) || 'james-hinton';
```

Comment:
Hard coded james-hinton. Do we want this?

Reply:
Yeah on the PR description, the workspaces endpoint is down, so have had to hard code it for now. Apparently will be fixed next sprint, so I put a function in there that when the functionality is implemented, it should work